docs-win/index.md

```diff
@@ -0,0 +1,297 @@
+
+![minidsp-rs site](./img/03.png)
+
+3\. Open the .tar.gz file with 7-Zip. The easiest method is to right click on the file, go to 7-Zip, and select Open archive.
+
+![Example decompression of the file](./img/04.png)
+
+4\. A .tar.gz file is actually 2 containers. The first one (the .gz) is now open in 7-Zip.
+
+![Example decompression of the file](./img/05.png)
+
+5\. Double-click on the file in the 7-Zip window, to open the .tar file.
+
+![Example decompression of the tar file](./img/06.png)
+
+6\. Highlight the minidsp.exe file, and click the Extract icon. Select a location for the file (such as the user's home directory) and make note of it as it will be needed for the ezbeq configuration.
+
+![Example extraction](./img/07.png)
+
+7\. To confirm basic functionality, open a Command Prompt.
+
+![Open a Command Prompt](./img/08.png)
+
+8\. In the Command Prompt window, type: `minidsp`
+
+![Example of minidsp output](./img/09.png)
+
+5\. If the MiniDSP 2x4HD is connected properly, similar information to the screenshot above will be displayed. Note the &quot;preset: 1&quot; in the image. That indicates that the MiniDSP 2x4HD is set to Config slot 2. The minidsp-rs application starts at 0 for the config presets, so 0 = Config slot 1, and so on.
+
+6\. If the MiniDSP 2x4HD is not detected, or is not connected, the following error will appear.
+![Example of minidsp error message](./img/10.png)
+
+### Install Chocolatey
+
+Chocolatey is a package manager for Windows, which will be used to install Python.
+
+1\. Open a browser and go to [https://chocolatey.org/install](https://chocolatey.org/install)
+
+![Example of installing python pre-requisites](./img/11.png)
+
+2\. Copy the command in the middle of the page. Heed the warnings about running commands directly from a website. This is not an endorsement of this site, but it is the simplest method of getting Python installed on Windows.
+
+The command (as of 2/10/21) is: `Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))`
+
+3\. Open a Powershell command with administrative privileges, otherwise the command will not work properly. This is done by finding Powershell from the Start menu, right-clicking on it, and selecting Run with Administrator privileges.
+
+![Powershell with admin privileges](./img/12.png)
+
+4\. Paste the command above into the Powershell window, and press enter.
+
+![Chocolatey installation](./img/13.png)
+
+5\. Close the Powershell window
+
+### Install Python
+
+ezbeq is built on Python, so it must be installed next.
+
+1\. Open a new Powershell window with administrative privileges (same process as above) and run the following:
+```
+cd ~
+Set-ExecutionPolicy -Scope CurrentUser
+RemoteSigned
+y
+
+choco install -y python --version=3.8.0
```

Comment:
3.8.0 is outdated, 3.8.7 is latest in the 3.8 line

Reply:
Valid point. By default, it tried to install 3.9. With 3.9, I received several errors.